client/extensions/woocommerce/app/settings/payments/index.js

```diff
@@ -51,17 +47,6 @@ class SettingsPayments extends Component {
 		className: PropTypes.string,
 	};
 
-	componentDidMount = () => {
-		const { site } = this.props;
-
-		// If we are in the middle of the Stripe Connect OAuth flow
-		// go ahead and option the Stripe dialog right away so
-		// we can complete the flow
-		if ( hasOAuthParamsInLocation() || hasOAuthCompleteInLocation() ) {
-			this.props.openPaymentMethodForEdit( site.ID, 'stripe' );
```

Comment:
Can you please also remove the `openPaymentMethodForEdit` prop and the `openPaymentMethodForEdit` function from `mapDispatchToProps`?

Reply:
Also done in https://github.com/Automattic/woocommerce-services/pull/2248/commits/ed7b1b779b6bed719596364d3ff8b45f3172768d